app/uk/gov/hmrc/claimvatenrolmentfrontend/config/AppConfig.scala

```diff
@@ -46,6 +46,8 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
   lazy val reportAProblemNonJSUrl = s"$contactFrontendUrl/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
   lazy val betaFeedbackUrl = s"$contactFrontendUrl/contact/beta-feedback?service=$contactFormServiceIdentifier"
 
+  lazy val tryAgainUrl = "/business-account/add-tax/vat/what-is-your-vat-number"
```

Comment:
I think this should have a clearer name like `businessTaxAccountAddVatUrl` or something to that effect